src/api/app/models/user.rb

```diff
@@ -842,11 +842,11 @@ def to_param
     to_s
   end
 
-  def nr_of_requests_that_need_work
-    Rails.cache.fetch("requests_for_#{login}", expires_in: 2.minutes) do
-      BsRequest.collection(user: login, states: %w(declined), roles: %w(creator)).count +
-      BsRequest.collection(user: login, states: %w(new), roles: %w(maintainer)).count +
-      BsRequest.collection(user: login, roles: %w(reviewer), reviewstates: %w(new), states: %w(review)).count
+  def tasks
+    Rails.cache.fetch("requests_for_#{cache_key}") do
+      declined_requests.count +
+      incoming_requests.count +
+      involved_reviews.count
```

Comment:
@adrianschroeter this is like that since more than 4 years:
https://github.com/openSUSE/open-build-service/commit/342da29d43f4c28941a69038976509b7f77a9fac

Main point is I guess: These requests are already covered in the outgoing requests table.

Git History:
342da29d43f4c28941a69038976509b7f77a9fac
d9b3878d42d530f6947357a87f50a02b44dedb71
f0017a174b968d8ff176728a336aa9ee8ab144ee
418365907d76011576ad3fc620814138f2133fe8
fe44dac14adb5d8ea55be3299585b27dcd3862ea